app/src/main/java/com/github/geohunt/app/model/database/firebase/FirebaseDatabase.kt

```diff
@@ -166,6 +171,16 @@ class FirebaseDatabase(activity: Activity) : Database {
         return dbUserRef.child(user.uid).setValue(userEntry)
     }
 
+    /**
+     * Point-Of-Interest user is a special user that does not have any information within the database
+     *
+     * It represent the user attach to any "public" challenge such as Point-Of-Interest. This design
+     * choices was made to have a location where we can fetch easily "all" point of interest
+     */
+    override fun getPOIUserID(): String {
+        return "0000000000000000000000000000"
```

Comment:
Why this? This looks very ugly (I mean, in my opinion). No need for 28 zeroes.

Reply:
It is to get more contribution on github. More seriously I think I just counted the number of letter in a UUID and replace them with 0. No particular other reason

Reply:
I know but wouldn't be better to just put a single 0? Because technically, those 28 zeroes can be a valid ID for an user (maybe Firebase Auth does not generate by default such an uid but better safe than sorry)

Reply:
No because nil UUID are a special uuid that can't be generated in UUID version 4 nor in UUID version 1

Reply:
Ok fair i guess. I've done it